src/CloudFoundry.Build.Tasks/src/BaseTask.cs

```diff
@@ -74,6 +78,8 @@ internal CloudFoundryClient InitClient()
                     }
                 }
 
+                this.CFPassword = this.CFPassword.Trim();
```

Comment:
I am not sure about this one. Can I have a space in my password in Cloud Foundry?